fenix/app/src/test/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckTelemetryMiddlewareTest.kt

```diff
@@ -234,4 +247,123 @@ class ReviewQualityCheckTelemetryMiddlewareTest {
 
         assertNotNull(Shopping.surfacePoweredByFakespotLinkClicked.testGetValue())
     }
+
+    @Test
+    fun `GIVEN a product review has been updated WHEN stale analysis is present and product is not in analysis THEN the stale analysis event is recorded`() {
+        val engineSession = mockk<EngineSession>()
+        val productTab = createTab(
+            url = "pdp",
+            id = "1",
+            isProductUrl = true,
+            engineSession = engineSession,
```

Comment:
❓ Do we need this? All we need is the tab url and make sure that's set as selected. Removing this will remove `mockk` from the tests.